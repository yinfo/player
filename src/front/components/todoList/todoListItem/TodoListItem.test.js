// @flow

import React from 'react';
import { shallow } from 'enzyme';
import TodoListItem from './TodoListItem';

describe('TodoListItem component', () => {
  it('renders as expected', () => {
    const props = {
      id: 1,
      label: 'label',
      done: false,
      statusLabel: 'statusLabel',
      statusLabelStyle: 'label-success',
      onListValidEdit: jest.fn(),
    };

    const component = shallow(<TodoListItem {...props} />);
    expect(component).toMatchSnapshot();
  });
});
