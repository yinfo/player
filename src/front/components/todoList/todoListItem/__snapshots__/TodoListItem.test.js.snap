// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoListItem component renders as expected 1`] = `ShallowWrapper {}`;
